### PR TITLE
Add troubleshooting information

### PR DESCRIPTION
**Pull request by lukeapage**
The following would have saved me several hours...

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -525,6 +525,21 @@ The changelog can be found in the [Changelog.md](Changelog.md) file.
 
 Firefox doesn’t allow cross-domain fonts: [Specifications](http://www.w3.org/TR/css3-fonts/#font-fetching-requirements), [Bugzilla Ticket](https://bugzilla.mozilla.org/show_bug.cgi?id=604421), [How to fix it](https://coderwall.com/p/v4uwyq).
 
+### My images are getting corrupted
+
+#### Using the node engine
+
+* Certain SVG's are not supported. See the [svg2ttf](https://github.com/fontello/svg2ttf) project which is used to convert from SVG to TTF (which is then converted forward to WOFF and WOFF2).
+* `autoHint` also adjusts the font file anc can cause your font to look different to the SVG so try switchining it off.
```
Comment (sapegin) [inline Readme.md]: anc = and? But disabling autoHint could also make icons look terrible on Windows.

**Comment by sapegin** [comment]
That’s very useful, thank you very much!

Related issue: #221.

**Comment by lukeapage** [comment]
Fixed spelling, added note about windows. For me (I'm on windows) it made many of my icons have corrupted paths, but that may be down to something else.

**Comment by sapegin** [comment]
:+1:

**Pull request merged**